go.mod

```diff
@@ -17,16 +17,19 @@ require (
 	github.com/gofrs/uuid v4.4.0+incompatible
 	github.com/google/go-containerregistry v0.20.3
 	github.com/hashicorp/go-multierror v1.1.1
+	github.com/joho/godotenv v1.5.1
```

Comment:
is this new? I cant see anything using it?

Reply:
used here https://github.com/kairos-io/AuroraBoot/blob/main/pkg/utils/common.go#L330